app/views/friendships/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<main>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <img src="/images/hogwarts_small.jpg" class="cover-photo hidden-xs hidden-sm img-responsive" alt="profile cover photo">
+        <div class="profile-photo">
+          <img src="/images/harry_potter_small.jpg" class="hidden-xs hidden-sm">
+        </div>
+      </div>
+    </div> <!-- end of row -->
+
+    <div class="row">
+      <%= render 'layouts/navbar' %>
+    </div> <!-- end of row -->
+
+    <section class="panel panel-default panel-custom">
+
+      <div class="panel-heading clearfix">
+        <% if !object_owner? %>
```

Comment:
You could have this entire thing (logic and link_to) in a helper to clean this up a bit